app/utils/whitelabel.ts

```diff
@@ -66,6 +69,9 @@ const SFFamilies: WhiteLabelSite = {
   logoLinkDestination: 'https://www.sffamilies.org/',
   navLogoStyle: styles.navLogoSFFamilies,
   showBanner: false,
+  showClinicianAction: false,
```

Comment:
I'm starting to think that we should have a base object with default values that we clone and modify for each of the individual sites, since it makes it difficult to see what the actual differences are between the sites when 90% of them are the same.

Just broke out https://github.com/ShelterTechSF/askdarcel-web/issues/1164 to capture this as a followup ticket.

Reply:
Thank you. Great idea!